doc/neptune-metadata.rst

```diff
@@ -84,7 +84,15 @@ Type
 Meaning
   Defines which types of auto-pull-requests the repository desires.
   Values can be specified for ``default`` and ``override``.
-  Possible values are ``none``, ``patch``, ``minor``, ``major``
+  Possible values are ``none``, ``patch``, ``minor``, ``major``.
+Default Value
+  If the repository is a library (``library: true``)
+  the default is ``patch``. Otherwise it will be ``minor``.
+
+  The motivation for this is, that for libraries the dependencies should be as
```

Comment:
Nitpick: I think the comma here (`this is, that`) is not needed.

Reply:
:+1: Mann mußt nicht Deutschkomma machen.

Reply:
JAWOHL